applications/nrf_desktop/doc/battery_meas.rst

```diff
@@ -0,0 +1,61 @@
+.. _battery_meas:
+
+Battery measurement
+###################
+
+The ``battery_meas`` module periodically measures the battery voltage and sends the ``battery_level_event`` event that informs about the current battery level.
+
+Module Events
+*************
+
++----------------+-------------+------------------+-------------------------+------------------+
+| Source Module  | Input Event | This Module      | Output Event            | Sink Module      |
++================+=============+==================+=========================+==================+
+|                |             | ``battery_meas`` | ``battery_level_event`` | :ref:`bas`       |
++----------------+-------------+------------------+-------------------------+------------------+
+
+Configuration
+*************
+
+The module uses the Zephyr :ref:`zephyr:adc` driver to measure voltage.
+For this reason, set the following options:
+
+* :option:`CONFIG_ADC` - the module implementation uses Zephyr ADC driver
+* :option:`CONFIG_ADC_ASYNC` - the module implementation uses asynchronous calls.
+* :option:`CONFIG_ADC_0` - the module implementation uses ``ADC_0``.
+* :option:`CONFIG_ADC_NRFX_SAADC` - the module implementation uses nrfx SAADC driver for nRF52 MCU series.
+
+Set ``CONFIG_DESKTOP_BATTERY_MEAS`` to enable the module.
+Also, make sure to define ``CONFIG_DESKTOP_BATTERY_MEAS_POLL_INTERVAL_MS``, that is the amount of time between the subsequent battery measurements (in ms).
+
+If the measurement circuit contains a voltage divider made of two resistors, set the ``CONFIG_DESKTOP_BATTERY_MEAS_HAS_VOLTAGE_DIVIDER`` option.
+The following defines specify the values of these two resistors (in kOhm):
+
+* ``CONFIG_DESKTOP_BATTERY_MEAS_VOLTAGE_DIVIDER_UPPER`` - upper resistor.
+* ``CONFIG_DESKTOP_BATTERY_MEAS_VOLTAGE_DIVIDER_LOWER`` - lower resistor.
+
+The module measures the voltage over the lower resistor.
+
+The battery voltage is converted to the state of charge percentage level units according to the lookup table defined in the ``battery_def.h`` file.
+You can find this file the in board-specific directory in the application configuration folder.
+
+Remember to also define the following battery parameters (otherwise, the default values will be used):
```

Comment:
I don't like how this requirement is hidden among explanations. I think we should find a better way to say what must be defned, and do it in one single place, but with explanations next to it... Cannot think of a way right now.

Reply:
If possible, I try to use bullet list with explainations next to parameters (if descriptions are short and simple).
In case the module has more advanced configuration options, I try to describe the options in groups. The options in every group are describing one subject, for example battery parameters. I think it is more understandable than putting all the options together in one list.
I think we have to assume that person who wants to configure module will read whole configuration section.